sys/net/gnrc/transport_layer/tcp/gnrc_tcp_fsm.c

```diff
@@ -33,6 +33,11 @@
 #include "debug.h"
 
 /**
+ * @brief Helper macro for LL_SEARCH to compare TCBs
+ */
+#define TCB_EQUAL(a,b)      ((a) == (b))
+
+/**
```

Comment:
In this context it is negligible if it is a function or a macro (have a look at the code), it just should return the right value, as you pointed out above (so this should be `!=` instead of `==`). However, in one point you are right, that usually the RIOT community prefers (inline) functions over macro magic ;-).